konami/products.md

```diff
@@ -3029,13 +3029,7 @@ List of misc/random stuff that could not be sorted properly so far, e.g. due to
 
 ### `LDJ-MA014-JA`
 
-`#WEB:https://mirais-station.hatenablog.com/entry/2020/01/23/180000`
-
-* Cannon Ballers/Rootage LCD
-* Model: Meiwa Denki ???
-* Size: 42"
-* Native resolution: ???
-* Input: DVI
+See dedicated [product doc](product/LDJ-MA014-JA.md)
```

Comment:
It's not that long. IDK if there is necessary to create a new file for it.

Reply:
Can't disagree with "it's not too long", but it feels more cleaned up to me this way to me. There are no conventions for this doc, but considering it's already quite bloated due to the simple fact that there are a lot of products out there, it might be a good idea moving products that have a bit more info (e.g. more than 2-3 lines?). Maybe a question to the maintainer @Shizmob.

Reply:
Awaiting for his code review.

Reply:
Sorry for let you waiting so long but seems he's not responding. I'll just merge it in.